axon-server/administration/reset-event-processor-token.md

```diff
@@ -0,0 +1,146 @@
+# How to reset the Token of an Event Processor
```

Comment:
Would it be feasible to rename this page to `event-processors.md`, so that we open up the file for other Event Processor administration sections later on? Just a section/page on resetting out of the blue feels slightly off to me  perhaps :thinking: